### DISCUSSION TITLE
Show variables read-only info

### DISCUSSION THREAD

**Discussion by kidonng** [PatrickF1/fzf.fish#191] [Ideas]
Fish recently added read-only info to `set --show` in https://github.com/fish-shell/fish-shell/commit/3db78232c6d5f32a2e8c1f08a4845712cce2b91b, however the way it displays is not covered by current regexes. I have some solutions in mind but they are not ideal, so I'm opening this discussion to see if someone have a better idea.

**Comment by PatrickF1** [comment]
Can you provide an example output that is problematic?

**Reply by kidonng** [reply]
I'm not testing the master branch fish but you can look at the linked commit, in `tests/checks/set.fish`, the `#CHECK` comments are checking for output, so the expected output will be something like:

```
$status: set in global scope, unexported, with 1 elements
Variable is read-only
$status[1]: |0|
```

**Reply by PatrickF1** [reply]
Interesting. I think we can either
1) Open an issue fish-shell to add $status to the "Variable is read-only line"
2) Just filter it out, maybe users don't care to have that info
